### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by Tokai5600** [misthero/dnd5e-custom-skills#43] [General]
Is there any plans on updating this module? Trying to use it with the current version of foundry and it just isn't working. Spent the past few days trying to get the custom skills to appear on the character sheets to no avail.

**Comment by misthero** [comment]
What version of foundry are you using?

**Reply by Tokai5600** [reply]
Version 12 stable, build 324

**Reply by Tokai5600** [reply]
Scratch that, we got it working!
